torch/fx/passes/backends/cudagraphs.py

```diff
@@ -21,15 +22,17 @@ def is_node_supported(self, submodules, node: torch.fx.Node) -> bool:
 
         found_not_cuda = False
 
+        FK = "val" if functorch.compile.config.use_fake_tensor else "fake_result"
```

Comment:
I think this should always just be "val" now.

Reply:
This will also fix the `aot_cudagraphs` 0% passrate observed in the nightlies.

Reply:
We should just fix all the uses of `fake_result` to use `val` instead.

Reply:
If we have `aot_cudagraphs` at 0% in nightlies we should temporarily disable the tests

Reply:
im just going to do a polyfill for now and we can update later